### PR TITLE
Rename MockServiceProvider::addDefinition() to setDefinition()

### PR DESCRIPTION
**Pull request by alien-art**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 ## 1.0.0 under development
 
 - Initial release.
+- Rename MockServiceProvider::addDefinition() to setDefinition()
```
Comment (vjik) [inline CHANGELOG.md]:
```suggestion
```

Not need for non-released packages.

**Review by vjik — CHANGES_REQUESTED** [review]
Thanks 👍
Need fix changlog only.

File: CHANGELOG.md (hunk 1 above)
Comment (alien-art) [reply CHANGELOG.md]: complete

**Review by xepozz — APPROVED** [review]

**Review by vjik — APPROVED** [review]
@xepozz We can merge this PR or need adapt anything package to this changes?

**Comment by xepozz** [comment]
> @xepozz We can merge this PR or need adapt anything package to this changes?

[We can merge it](https://github.com/search?q=org%3Ayiisoft+addDefinition&type=code)

**Pull request merged**